contracts/scripts/interactions/InteractRiverRegistry.s.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+// interfaces
+import {IRiverConfig} from "contracts/src/river/registry/facets/config/IRiverConfig.sol";
+
+// libraries
+
+// contracts
+import {Interaction} from "contracts/scripts/common/Interaction.s.sol";
+
+contract InteractRiverRegistry is Interaction {
+  function __interact(address deployer) internal override {
+    address riverRegistry = getDeployment("riverRegistry");
+
+    uint64[] memory chains = new uint64[](13);
+    chains[0] = 85432;
```

Comment:
Can you put a comment after the chain ids with the network they reference?

e.g.

```
chains[0] = 85432/ // Base Sepolia
...
chains[8] = 10; // Gnosis Mainnet
```

Reply:
fixed in latest push